addons/web/static/src/views/relational_model/dynamic_record_list.js

```diff
@@ -87,13 +87,14 @@ export class DynamicRecordList extends DynamicList {
     }
 
     _createRecordDatapoint(data, mode = "readonly") {
+        const { context, activeFields, resModel, fields } = this.config;
```

Comment:
In the datapoint, we clone this.fields.
So I have to at least pass the fields from the config. Shouldn't id and display_name be added when defining the config in the relational_model? To avoid making a clone of fields?